vyper/codegen/external_call.py

```diff
@@ -71,7 +71,7 @@ def _pack_arguments(fn_type, args, context):
     pack_args.append(["mstore", buf, util.method_id_int(abi_signature)])
 
     if len(args) != 0:
-        pack_args.append(abi_encode(add_ofst(buf, 32), args_as_tuple, context, bufsz=buflen))
+        pack_args.append(abi_encode(add_ofst(buf, 32), args_as_tuple, context, bufsz=buflen - 32))
```

Comment:
too much going on in this line, let's split across multiple lines
```suggestion
        new_buf = add_ofst(buf, 32)
        new_buflen = buflen - 32
        pack_args.append(abi_encode(new_buf, args_as_tuple, context, bufsz=new_buflen))
```